.github/workflows/auto_merge.yaml

```diff
@@ -3,12 +3,11 @@ name: Dependabot auto-merge
 on:
   pull_request:
 
-permissions:
```

Comment:
For top level permissions, I was told to set “permissions: read-all” as the default (in case the org sets something different, this forces read only)

Reply:
we already haver a global setting in the org config to set to read all as default, but can add it here if you want as well